tests/test_sources.py

```diff
@@ -0,0 +1,49 @@
+#
+# test_sources.py
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Tests for the repository sources files
+#
+
+
+import os
+import re
+
+
+def test_paths_matches_install(apt_sources_install):
+    from kano_updater.paths import KANO_SOURCES_LIST
+
+    assert os.path.exists(KANO_SOURCES_LIST)
```

Comment:
- Let's move REFERENCE_STRETCH_LIST from scenarios.py (as well as other appropriate ones) into paths.py
 - Let's add another assert for REFERENCE_STRETCH_LIST file such that we know the symlink isn't broken

Reply:
Moved the sources dir, `OS_SOURCES_REFERENCE` and `REFERENCE_STRETCH_LIST` to `paths.py` and added a test assertion that the references exist.